### PR TITLE
Attempt fix autograding json

### PR DESCRIPTION
**Pull request by Waqar144**
For merely setting up a simple repo like this, I don't think you need a `.yml` file. At least the [official docs](https://classroom.github.com/help/adding-tests-for-auto-grading) don't mention it.

The command in "run" was not running anything, so I have fixed that. It may still fail because of a mismatch in expected vs actual but that will be trivial to fix.

----

**PS** As far as i can see, you can not set up "autograding" with MS-Unit-Test-Suite. The ideology here is totally different from the normal workflow setup. But if you still need to make those work, you can do so using github actions.

@Maleehak

### CODE REVIEW

**Comment by Maleehak** [comment]
yes i tried to write the ./output but it showed errors. Can we write all the test cases like this? for functions/structures? or is it better to add separate test file and then run it using github actions? @Waqar144

**Comment by Waqar144** [comment]
@Maleehak 
> yes i tried to write the ./output but it showed errors.

The error now is just wrong output from the program (In case this was a submission from a student, this tells us that the student's output was incorrect).

> Can we write all the test cases like this? for functions/structures? or is it better to add separate test file and then run it using github actions? @Waqar144

I haven't ever personally dealt with this "autograding" feature of github, this is my first exposure to it. The advantages are obvious:
- You don't have to write code etc. Just specify the output you expect from the students program


Adding a separate test file is also okay, I am more used to this personally and this is what the industry does as well. It also depends on what the assignment is. As an example, consider the assignment is to implement a "LinkedList" with functions `add(), remove(), isEmpty(), size()`. In this case, using custom test cases is much better I think and it allows for more finegrained testing.

- Basically, you will just write some test cases in a tests file and store it here on the github.
- The students need to name the file as specified E.g `List.h / List.cpp`
- The functions of the assignment MUST be exactly as specified by the teacher or the code will fail to compile
- When a student creates a PULL Request, the tests will automatically run and check the output for each function and finally report failed / passed cases.

Also, I recommend using "Catch2" library for testing as it will  be much easier to setup and use.

Note that for simple programs, such as the one in this repository, using a test library / framework is kind of impossible as there are no functions to test. In this case, using the Github autograding feature is the only simple solution.

**Comment by Maleehak** [comment]
Yes autograding system is the latest for Github classroom. that's why there is very little documentation about it. I will explore Catch2 library now.
Thanks a lot

**Comment by Waqar144** [comment]
@Maleehak No issues, feel free to ping me anytime and I will see if I can help
